File: codeforces/96.restore_the_permutation_by_merger_[1385B]/rtpbm.py
# Author: José Rodolfo (jric2002)
t = int(input())
while (t):
  n = int(input())
  a = list(map(int, input().split(" ")))
  p = list()
  while (bool(len(a))):
    p.append(a[0])
    a.pop(a.index(a[0], 1))
    a.pop(0)
  for e in p:
    print(e, end=" ")
  print(end="\n")
  t -= 1

File: codeforces/low_difficulty/800_points/96.restore_the_permutation_by_merger_[1385B]/rtpbm.cpp
#include <iostream>
#include <vector>
#include <algorithm>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  unsigned short int t, n, ai;
  vector<unsigned short int> a;
  vector<unsigned short int> p;
  cin >> t;
  while (t--) {
    cin >> n;
    for (unsigned short int i = 0; i < (n * 2); i++) {
      cin >> ai;
      a.push_back(ai);
    }
    while (!a.empty()) {
      p.push_back(a.at(0));
      a.erase(find(a.begin() + 1, a.end(), a.at(0)));
      a.erase(a.begin());
    }
    for (auto e : p) {
      cout << e << " ";
    }
    cout << endl;
    a.clear();
    p.clear();
  }
  return 0;
}